src/features/ExtensionCommands.ts

```diff
@@ -161,11 +161,8 @@ export class ExtensionCommandsFeature extends LanguageClientConsumer {
 
             vscode.commands.registerCommand("PowerShell.InvokeRegisteredEditorCommand",
                 async (param: IInvokeRegisteredEditorCommandParameter) => {
-                    if (this.extensionCommands.length === 0) {
-                        return;
-                    }
-
-                    const commandToExecute = this.extensionCommands.find((x) => x.name === param.commandName);
+                    const commandToExecute = this.extensionCommands.find(
```

Comment:
Just removed a pointless length check. It's not undefined so `.find` is just going to return nothing, which is later checked anyway.